### ISSUE TITLE
Question about salmon/sample/*quant.sf file in scripts

### ISSUE THREAD

**Issue by NJU-Bio-Info** [mfansler/utr-enhancers-pipeline#1]
Hi, when I learn your R scripts about DESeq and DEXSeq, I wonder that the quant.sf file here is the output of qapa?

**Comment by mfansler** [comment]
Thank you for the question. The `.sf` file is the output from `salmon`. For example, [the rule `salmon_quant_pe`](https://github.com/Mayrlab/utr-enhancers/blob/8c93dd997fd3738cc85bde280218424a9dc9f844/Snakefile#L128) will generate this by running the `salmon quant` command.

I will add a README file in the next days, to describe more in detail how the pipeline is run. Note that what is currently [the `normalize`](https://github.com/Mayrlab/utr-enhancers/tree/normalize) is what was used in the final analysis. I will eventually move this too `main`. I'll let you know when I've updated, but free feel to ask any more clarifications!

**Comment by NJU-Bio-Info** [comment]
Yeah, I really want to learn from your scripts about how to identify different polyA sites usage(or transcript isoform) with the output of QAPA. If you can make some clarification for this topic, you will help me a lot about my project!!! Really want to see your update!

**Comment by NJU-Bio-Info** [comment]
The index for salmon used to generating .sf file in DEXseq.R and DESeq.R is the output utr sequence(.fa file)? or the whole genome(transcriptome)?

**Comment by mfansler** [comment]
That's a good question. I'll make sure to include the provenance for that in the write-up. 

The index is from [the QAPA v1.3.0 release](https://github.com/morrislab/qapa/releases/tag/v1.3.0), then using the UCSC mm10 genome as a decoy. The steps to replicate the index are documented in [the Snakefile](https://github.com/mfansler/mm10-db/blob/220f75fa8b6882df6f1f9e101bbf7d91e0c254fa/Snakefile#L165) in my `mm10-db` repository.
